### PR TITLE
added rubocop yml and cleaned up style issues

### PR DESCRIPTION
**Pull request by sibsmc**
Added rubocop yml and made style changes to remove issues found by rubocop

### CODE REVIEW

File: ui/build/asset-manifest.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "static/css/main.12cb9920.chunk.css.map": "/static/css/main.12cb9920.chunk.css.map",
+    "static/js/2.180d86fb.chunk.js.LICENSE.txt": "/static/js/2.180d86fb.chunk.js.LICENSE.txt"
+  },
+  "entrypoints": [
+    "static/js/runtime-main.83c3e0c4.js",
+    "static/js/2.180d86fb.chunk.js",
+    "static/css/main.12cb9920.chunk.css",
+    "static/js/main.d6f017de.chunk.js"
+  ]
+}
```
Comment (olleolleolle) [inline ui/build/asset-manifest.json:22]: This looks like it ought not to be included in this PR.

And that goes for all of the `ui/build` directory. Perhaps that directory is a candidate for the gitignore file?

**Review by olleolleolle — CHANGES_REQUESTED** [review]
👋 Hi! Super initiative, to add an automated style linter, keeping all the files looking about the same.

There may be an undesired addition here: the changes to the `ui` directory seem unrelated to this change (and are probably not to be added to the repository).

I got suspicious when I saw 4million+ lines are added. "RuboCop can't add _that_ many lines", I thought.

File: ui/build/asset-manifest.json (hunk 1 above)
Comment (sibsmc) [reply ui/build/asset-manifest.json:22]: I don't know how i did this, build and node_modules are both in gitignore, but for some reason they are all being pulled in

Comment (sibsmc) [reply ui/build/asset-manifest.json]: I was worried i somehow managed to copy my whole home directory in :sweat_smile:

**Review by olleolleolle — APPROVED** [review]
Looking grand!

**Pull request merged**